src/AkeneoPimClientBuilder.php

```diff
@@ -146,6 +150,20 @@ public function buildAuthenticatedByToken(string $clientId, string $secret, stri
         return $this->buildAuthenticatedClient($authentication);
     }
 
+    public function enableCache(): self
```

Comment:
what is the default behavior?

Reply:
Disable by default to avoid BC Break

Reply:
Adding a comment could be useful for DX I think for quick access to this information